architecture-decision-records/operator/ODH-ADR-Operator-0011-Perses-dashboard-guidelines.md

```
@@ -0,0 +1,207 @@
+
+### Architecture Overview
+
+The Perses dashboard architecture in RHOAI consists of:
+
+1. **Component Level**: Teams create `PersesDashboard` and `PersesDatasource` CRs
+2. **Operator Level**: OpenDataHub operator deploys and manages Perses resources
+3. **Platform Level**: Perses platform renders dashboards from CR specifications
+4. **Data Layer**: Datasources query Prometheus/Tempo for metrics and traces
+5. **UI Integration**: Dashboard UI displays Perses dashboards (RHAISTRAT-113)
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)